### PR TITLE
Duet-song-test

### PR DESCRIPTION
**Pull request by RenBabiuch**

### CODE REVIEW

File: src/Scenes/Game/Singing/GameOverlay/Components/Lyrics.tsx [hunk 1]
```diff
@@ -212,6 +212,7 @@ const BasePassTheMicProgress = styled.div<{ color: string }>`
 
 const PassTheMicProgress = (props: { progress: number } & ComponentProps<typeof BasePassTheMicProgress>) => (
   <BasePassTheMicProgress
+    data-test="pass-the-mic"
```
Comment (Asvarox) [inline src/Scenes/Game/Singing/GameOverlay/Components/Lyrics.tsx:215]: lets call it `pass-the-mic-progress`

File: tests/duet-song.spec.ts [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+const player2 = 1;
+const player1Name = 'all-Karaoke';
+const polishLang = 'Polish';
+const duetPolSong = 'e2e-multitrack-polish-1994';
+const polArtist = '2 E2ETest';
+const polSongTitle = 'Multitrack';
+const duetsPlaylist = 'Duets';
+
+test('Sing a song intended for a duet as a single player', async ({ page, browserName }) => {
+  test.fixme(browserName === 'firefox', 'Test fails super often on FF');
```
Comment (Asvarox) [inline tests/duet-song.spec.ts:22]: I think the description here is not right

File: tests/duet-song.spec.ts [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+});
+
+const duetSpanSong = 'e2e-pass-test-spanish-1994';
+const artist = 'E2E-el-Dueto';
+const songTitle = 'Pass Test';
+const spanishLang = 'Spanish';
+const gameMode = 'Pass The Mic';
+
+test('Sing a duet song in pass-the-mic mode as a single connected player', async ({ page, browserName }) => {
+  test.fixme(browserName === 'firefox', 'Test fails super often on FF');
```
Comment (Asvarox) [inline tests/duet-song.spec.ts:91]: (same here)

File: tests/duet-song.spec.ts [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  });
+
+  await test.step('Score should be visible for player1', async () => {
+    await expect(pages.gamePage.getPlayerScoreElement(player1)).toBeVisible();
+    await expect(pages.gamePage.getPlayerScoreElement(player2)).not.toBeVisible();
+  });
+
+  await test.step('Pass the mic progress and change indicator icon should be displayed', async () => {
+    await expect(pages.gamePage.passTheMicProgressElement).toBeVisible();
+    // await expect(pages.gamePage.changeIndicatorIcon).toBeVisible({ timeout: 20_000 });
```
Comment (Asvarox) [inline tests/duet-song.spec.ts:150]: lets create a separate issue to fix this

Comment (RenBabiuch) [reply tests/duet-song.spec.ts:150]: Ok, done #207 :)

**Pull request merged**